save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/utils/KotlinUtilsJvm.kt

```diff
@@ -7,3 +7,4 @@ package com.saveourtool.save.utils
 import org.springframework.http.ResponseEntity
 
 typealias EmptyResponse = ResponseEntity<Void>
+typealias StringResponse = ResponseEntity<String>
```

Comment:
we have `StrungResponse` in `com/saveourtool/save/backend/SaveApplication.kt:11`, can you please replace old one by a new one introduced by you?

Reply:
Sure

Reply:
Done!